app-engine/java/gradle/src/com/google/cloud/tools/intellij/appengine/java/gradle/AppEngineGradleProjectDataService.java

```diff
@@ -63,15 +63,17 @@ public void importData(
         .forEach(
             module -> {
               AppEngineGradleModule appEngineGradleModule = moduleNameToModel.get(module.getName());
-              if (PlatformUtils.isIdeaCommunity()
```

Comment:
the idea here is that previously the data fetched from the Gradle model was only persisted when the app-engine-gradle-plugin was available (since in the previous use-case, this was the only time we needed it). Now we need this info for other purposes, so I removed the requirement that the plugin be available.

Similarly, if you see in `AppEngineGradleDeploymentSourceProvider` I added the check for the plugin there instead. 

In other words, the data from the Gradle model now is always available. Instead I moved the check for the app-gradle-plugin from sync-time to deployment source addition time.